drivers/sensor/bm160/Kconfig

```diff
@@ -208,4 +216,88 @@ config BMI160_GYRO_ODR_3200
 	bool "3200 Hz"
 endchoice
 
-endif # BMI160
+config BMX160_MAG
```

Comment:
This option is not available for `I`, but it will still be shown on it.

Reply:
No, config has depends on BMX160 so it will not shown to BMI160